internal/cli/atlas/search/index_opts.go

```diff
@@ -86,28 +86,28 @@ func (opts *IndexOpts) newSearchIndex() (*atlas.SearchIndex, error) {
 	if err != nil {
 		return nil, err
 	}
-	i := &atlas.SearchIndex{
-		Analyzer:       opts.analyzer,
+	i := &atlasv2.FTSIndex{
+		Analyzer:       &opts.analyzer,
 		CollectionName: opts.collection,
 		Database:       opts.dbName,
-		Mappings: &atlas.IndexMapping{
-			Dynamic: opts.dynamic,
-			Fields:  &f,
+		Mappings: &atlasv2.FTSMappings{
+			Dynamic: &opts.dynamic,
+			Fields:  f,
 		},
 		Name:           opts.name,
-		SearchAnalyzer: opts.searchAnalyzer,
+		SearchAnalyzer: &opts.searchAnalyzer,
```

Comment:
🥇 
I like this approach
Let's try to amend the input model like this where possible for the "Atlas only" commands @swattyT @andmatei